rails/config/routes.rb

```diff
@@ -1,7 +1,7 @@
 # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 Rails.application.routes.draw do
   scope '(:locale)', locale: /#{I18n.available_locales.join("|")}/ do
-    scope '/member', module: 'dashboard', constraints: RoleRoutingConstraint.new { |user| !user.super_admin } do
+    scope '/member', module: 'dashboard', constraints: RoleRoutingConstraint.new do
```

Comment:
Just want to call out that this change makes the assumption that a `super_admin` should be able to access every page, which seems reasonable to me but maybe was intentionally restricted in the past for a reason we're not aware of.